cli/src/semdep/parsers/util.py

```diff
@@ -214,17 +215,18 @@ def parse_error_to_str(e: ParseError) -> str:
 def safe_path_parse(
     path: Optional[Path],
     parser: "Parser[A]",
-    preprocess: Optional[Callable[[str], str]] = None,
+    preprocess: Callable[[str], str] = lambda ξ: ξ,  # ξ kinda looks like a string hehe
```

Comment:
defaulting to a no-op lambda means we get to remove some branching in the code

Reply:
🛐 unicode 🛐